src/app/metadata/features/dataset-details/dataset-details.html

```diff
@@ -238,19 +238,31 @@ <h2 class="inline-flex items-center text-2xl">
                 </p>
                 <p>
                   <strong>Data Use Modifiers:</strong>
-                  @if (dap().data_use_modifier_terms.length > 0) {
-                    @for (
-                      term of dap().data_use_modifier_terms;
-                      track term;
-                      let idx = $index;
-                      let first = $first;
-                      let last = $last
-                    ) {
-                      {{ first ? ' ' : '' }}
-                      <a appExtLink [href]="dap().data_use_modifier_urls[idx]">{{
-                        term
-                      }}</a
-                      >{{ last ? '' : ', ' }}
+                  @if (dap().data_use_modifier_ids && dap().data_use_modifier_terms) {
```

Comment:
The computed signal could prepare the data already so that only single check would be needed here (not separate ones of ids and terms, and null and length).